src/web/lib/dailyArticleCount.ts

```diff
@@ -0,0 +1,76 @@
+interface DailyArticle {
+    day: number;
+    count: number;
+}
+
+export type DailyArticleCount = Array<DailyArticle>;
+
+// in localStorage, has format {"value":[{"day":18459,"count":1},{"day":18457,"count":1},{"day":18446,"count":1}]} to match frontend
+interface DailyArticleCountLocalStorage {
```

Comment:
interface which determines the format that dailyArticleCount should be stored in localStorage as